WordPress/Classes/ViewRelated/Pages/PageListTableViewCell.m

```diff
@@ -114,14 +140,41 @@ - (void)configureBadges
 {
     Page *page = (Page *)self.post;
 
-    if (page.hasPendingReviewState) {
-       self.privateBadgeLabel.text = NSLocalizedString(@"Pending review", @"Title of the Pending Review Badge");
+    NSString *badgesString = @"";
+    BOOL displayFirstBadge = page.hasPrivateState || page.hasPendingReviewState;
```

Comment:
I wonder if the logic in this method could be simplified.  
Currently its one conditional check to assign the value of displayFirstBadge, then another check to see if we need to assign a string value, then another check to determine if the dash spacer.
Would it be nicer to do something like this? 
``` 
if (page.hasPendingReviewState) {
  // assign string
} else if (page.hasPendingReviewState) {
  // assign string
}

if (page.hasLocalChanges) {
  if ( badgeString.length > 0) {
    // append string
  }
  // append string
}
```
It would avoid the need to have `displayFirstBadge`.   Thoughts?

Reply:
Yep! Much better!